weld/common/src/main/java/org/jboss/as/weld/interceptors/Jsr299BindingsInterceptor.java

```diff
@@ -41,9 +41,9 @@
 import org.jboss.weld.ejb.spi.InterceptorBindings;
 
 /**
- * Interceptor for applying the JSR-299 specific interceptor bindings.
+ * Jakarta Interceptors for applying interceptors bindings.
  * <p/>
- * It is a separate interceptor, as it needs to be applied after all
+ * It is a separate interceptors, as it needs to be applied after all
```

Comment:
s/It is a separate interceptors/It is a separate interceptor

Reply:
Thanks! Done :)